src/unix/linux_like/linux/gnu/b64/powerpc64/mod.rs

```diff
@@ -12,6 +12,8 @@ pub type suseconds_t = i64;
 pub type __u64 = ::c_ulong;
 
 s! {
+    pub struct pt_regs {}
```

Comment:
I don't know how to handle the fieldless C struct case, the test results shows that there are still discrepancies between Rust and C:

```
bad pt_regs size: rust: 0 (0x0) != c 352 (0x160)
bad pt_regs align: rust: 1 (0x1) != c 8 (0x8)
size of struct pt_regs is 352 in C and -980681736 in Rust
```

Any help is greatly appreciated

Reply:
@gnzlbg Wondering if you could give me a bit of guidance here. Many thanks in advance

Reply:
@JohnTitor Wondering if you could give me a bit of guidance here. Many thanks in advance

Reply:
I believe this commit has some context: https://github.com/bminor/glibc/commit/d57cb31910ca5c200e4172276749a7f8bd17ae3c
It says `struct pt_regs (not defined in this header)` so I guess we should refer to Linux source instead?
https://github.com/torvalds/linux/blob/c48b75b7271db23c1b2d1204d6e8496d91f27711/arch/powerpc/include/asm/ptrace.h#L26-L63